source/Tests/Tests.csproj

```diff
@@ -20,6 +20,8 @@
     <PackageReference Include="Microsoft.CodeAnalysis" Version="3.8.0" />
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Analyzer.Testing.NUnit" Version="1.0.1-beta1.21064.2" />
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp.CodeFix.Testing.NUnit" Version="1.0.1-beta1.21064.2" />
+    <PackageReference Include="xunit.assert" Version="2.4.1" />
+    <PackageReference Include="xunit.core" Version="2.4.1" />
```

Comment:
Need to add these ones in so we can reference these types in the unit tests. Haven't converted anything to xUnit... _though I would if nobody had a problem with it_

Reply:
I don't see why not.

Reply:
Switch away (to XUnit). I would lean towards FluentAssertions instead of XUnit.Assert

Reply:
Don't actually need the Assert library was in here anyway, so I've been able to pull that out and these all still pass.

I'll do an xUnit switcheroo as a separate piece. Should be easy enough. 👌